admin-dev/themes/new-theme/js/app/pages/logs/index.js

```diff
@@ -90,5 +92,13 @@ class LogsPage {
 }
 
 $(() => {
+  // arrow function is incompatible with availability of $(this).
+  $('.reset-search').on('click', function () {
+      let url = $(this).data('url');
```

Comment:
use `const` instead

Reply:
why?

Reply:
You don't change the value of url. `const foo = {};` is still better then `let foo = {}` since you cannot re-assign variable afterwards.